### PR TITLE
Simple android display

### PR DESCRIPTION
**Pull request by ZappoMan**
test code for Chris
- Implements Basic 2D window for android
- Implements mouse/keyboard input for android (touch events work!)
- supports both 2D "Desktop" display and VR "Daydream" display

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: e431bd054ab48ac6a9979978f310ddf93c723607 available at: https://gist.github.com/8d227214e995e2fdd0464f6271846278<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10508-e431bd0.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10508-e431bd0.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 721a5d7c8f7d011f84922077b2003cb80069be3e available at: https://gist.github.com/8d227214e995e2fdd0464f6271846278<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10508-721a5d7.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10508-721a5d7.dmg

File: libraries/display-plugins/src/display-plugins/DisplayPlugin.cpp [hunk 1]
```diff
@@ -24,25 +24,34 @@ const QString& DisplayPlugin::MENU_PATH() {
 }
 
 // TODO migrate to a DLL model where plugins are discovered and loaded at runtime by the PluginManager class
+
 DisplayPluginList getDisplayPlugins() {
+    qInfo() << __FUNCTION__;
```
Comment (jherico) [inline libraries/display-plugins/src/display-plugins/DisplayPlugin.cpp]: does this debug code need to stay?

File: libraries/display-plugins/src/display-plugins/OpenGLDisplayPlugin.cpp [hunk 2]
```diff
@@ -605,19 +604,26 @@ void OpenGLDisplayPlugin::compositeLayers() {
...
         batch.enableStereo(false);
         batch.resetViewTransform();
         batch.setFramebuffer(gpu::FramebufferPointer());
         batch.setViewportTransform(ivec4(uvec2(0), getSurfacePixels()));
-#ifndef ANDROID
-        batch.setResourceTexture(0, _compositeFramebuffer->getRenderBuffer(0));
+
+#if HAS_COMPOSITE_FRAME_BUFFER //ndef ANDROID
+        //batch.setResourceTexture(0, _compositeFramebuffer->getRenderBuffer(0));
+        batch.setResourceTexture(0, _currentFrame->framebuffer->getRenderBuffer(0));
```
Comment (jherico) [inline libraries/display-plugins/src/display-plugins/OpenGLDisplayPlugin.cpp]: why are both blocks of this condition the same?  Shouldn't this block be `batch.setResourceTexture(0, _compositeFramebuffer->getRenderBuffer(0));`

File: libraries/display-plugins/src/display-plugins/DisplayPlugin.cpp (hunk 1 above)
Comment (ZappoMan) [inline libraries/display-plugins/src/display-plugins/DisplayPlugin.cpp]: this PR is a hacked demo to help Chris get a demo going... it's not ready for merging

File: libraries/display-plugins/src/display-plugins/OpenGLDisplayPlugin.cpp (hunk 2 above)
Comment (ZappoMan) [inline libraries/display-plugins/src/display-plugins/OpenGLDisplayPlugin.cpp]: this PR is out of date - closing

**Pull request closed without merging**